driver/normalizer/strconv_test.go

```diff
@@ -17,16 +17,23 @@ var testCasesUnquote = []struct {
 	// to be decoded via Go's native rules to the byte sequence we want.
 	canonicalQuoted string
 }{
-	{`'a'`, "a", `'a'`},
-	{`'\x00'`, "\u0000", `'\x00'`},
+	{`'a'`, "a", "'a'"},
+	{`'\x00'`, "\u0000", "'\\x00'"},
 	{`'\0'`, "\u0000", "'\\x00'"},
 	{`'\0something\0'`, "\u0000something\u0000", "'\\x00something\\x00'"},
 	{`'\0something\0else'`, "\u0000something\u0000else", "'\\x00something\\x00else'"},
 	{`'\u0000123\0s'`, "\u0000123\u0000s", "'\\x00123\\x00s'"},
+	{`'\0\0'`, "\u0000\u0000", "'\\x00\\x00'"},
+}
+
+func TestUnquoteDouble(t *testing.T) {
+	s, err := unquoteDouble(`"\0\0\0\0\0\0\0\0"`)
```

Comment:
May we also have a case where the string includes non-NUL runes?

Reply:
👍 but further improvement for this case are going to be part of subsequent PRs under the same ☔️ issue.